packages/components/src/vue/vuex-modules/data/lexis.js

```diff
@@ -75,6 +76,10 @@ export default class Lexis extends Module {
     return Boolean(this._lexisConfig && this._lexisConfig.cedict)
   }
 
+  hasLexiconsConfig () {
+    return Boolean(this._lexiconsConfig)
```

Comment:
Should we also check that it is not an empty object?

Reply:
I don't know. I think it's okay if it's an empty object, as it will have no impact and is a possible way of saying there are no remote settings available.

Reply:
I think that makes sense. I think we can consider an empty config object as: "there is a configuration but it has no config items in it". Maybe we can document it somewhere.

Reply:
I will note that.